src/sqlfluff/core/parser/segments/raw.py

```diff
@@ -177,16 +177,16 @@ def raw_trimmed(self):
             return raw_buff
         return raw_buff
 
-    def raw_list(self):  # pragma: no cover TODO?
+    def raw_list(self) -> list:  # pragma: no cover TODO?
         """Return a list of the raw content of this segment."""
         return [self.raw]
```

Comment:
This method can just be deleted. It's not used anymore. It has a counterpart in `BaseSegment` which should also go. They're replaced by `.raw_segments`.